_rules/status-text-has-aria-live--mogq50.md

```diff
@@ -0,0 +1,73 @@
+---
+id: mogq50
+name: 'Status text update has `aria-live` property'
+rule_type: atomic
+description: |
+  This rule checks that any text update that meets the definition of a [status message][] has `aria-live` property.
+accessibility_requirements:
+  wcag21:4.1.3: # Status Messages (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:ARIA22: # Using role=status to present status messages
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:ARIA19: # Using ARIA role=alert or Live Regions to Identify Errors
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility Tree
+acknowledgments:
+  authors:
+    - Aron Janecki
+    - Jean-Yves Moyen
+---
+
+## Applicability
+
+This rule applies to any [HTML element][] that has a [text node][] as a [descendant][] in the [flat tree][] if:
+
+- **changed**: the `innerText` property of the element changes; and
+- **available**: the [text node][] is [included in the accessibility tree][].
+
+## Expectation
+
+Each test target has an implicit or explicit `aria-live` property value that is [ASCII lowercase][] match for string "assertive", "polite", or "off".
```

Comment:
As far as I understand, `off` being the default value means that the attribute value of `aria-live` is always going to be one of these three (since that is the only possible values).

Reply:
Hmm, not necessarily, the implicit value of `role="alert"` is "assertive"` which I consider to be the default value. The idea here was to make sure that `aria-live` property has one of the allowed values, as you mentioned, but also to make sure that cases, where updates are turned off by an instrument, are not failing the rule. That is, when there is a button that toggles between "off" and "polite" depending on the context, just like the [ARIA 1.2 carousel example](https://www.w3.org/TR/wai-aria-practices-1.2/examples/carousel/carousel-1.html).

Reply:
But `aria-live` only allows values `assertive`, `polite` or `off`. So it is not possible to fail this expectation.

Reply:
Ahh, of course, the word property is already implying that only one of the allowed values can be used as the attribute value.  I think this expectation is ok. Do you recommend adding something about the default value or shortening it only to "Each test target has an implicit or explicit `aria-live` property"?

Reply:
🤔 Maybe I just need to see a Failing Example for the rule. I'm currently unable to build one…

Reply:
If we take the following as an example
```
<div role="status">10 results out of 5</div>
```

When I'm using aViewer to see which properties are added as an implicit `aria-live` value for the above element, in IAccessible2, I can see `container-live: polite`, which is correct. The same works for `role=timer`. The `role="alert` does not have the `container-live` property (just as I read in the link you sent- thanks!).

When you add the  `aria-live="politeness"` to the above example, the `container-live` property value is `politeness`. This is a failure of the rule.

Should we be using the word property? I mean, platform accessibility API's do not have such a property (the equivalent of `aria-live`) so it is conveyed as an object attribute pair. The thing that is currently confusing in my opinion, and I might be mistaken here, is that  `container-live` is a property of the Object Attributes construct.  I guess we could, only if we defined it well. This would also future proof the rule when platform accessibility API's develop an equivalent of `aria-live`.

Also, sorry about the confusion with `off` being the default value. In my explanation, I completely forgot about the spec and was thinking about something else. I don't think we should add anything there that would say `off` is the default value. I've tried a couple of times and `container-live` is only showing `off` value when it's explicitly specified.

Reply:
Disregard my comment about the `off` being default! I think that we need to ditch the `off` value. 
So, in the following example:
```
<div role="status" aria-live="politeness">Status message 1 of 10</div>
```

When checking UIA in Inspect using Chrome, there is no `LiveSettingProperty`. As soon as `aria-live=politeness"` is removed, the `LiveSettingProperty` appears and it has the value of `Polite`, which is correct. 

When doing the same in Edge, the `LiveSettingProperty` has value `Off` right from the start.

I think that removing `off` from the expectation will provide better results for the rule.